app/views/layouts/_navigation.html.erb

```diff
@@ -1,48 +1,65 @@
-<nav class = "navbar navbar-expand-lg bg-body-tertiary">
-  <div class = "container-fluid">
-    <a class = "navbar-brand" href="<%= root_path %>">Cinema</a>
-    <button class = "navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
-      <span class = "navbar-toggler-icon"></span>
-    </button>
-    <div class = "collapse navbar-collapse" id="navbarSupportedContent">
-      <ul class = "navbar-nav me-auto mb-2 mb-lg-0">
-
-          <% if !logged_in? %>
-              <li class = "nav-item">
-              <a class = "nav-link" aria-current="page" href="<%= login_path %>">Login <i class="fa-solid fa-right-to-bracket"></i></a>
-            </li>
-            <li class = "nav-item">
-              <a class = "nav-link" href="<%= new_user_path %>">Sign up <i class="fa-regular fa-user-plus"></i></a>
-            </li>
-          <% else %>
-            <li class = "nav-item">
-              <div class = "logout-div"><%= button_to "log out ",logout_path,class:"logout-btn",method: :delete %><i class="fa-solid fa-right-from-bracket"></i></div>
-            </li>
+<nav class="navbar navbar-expand-lg bg-body-tertiary">
+  <div class="container-fluid">
+    <a class="navbar-brand" href="<%= root_path %>">Cinema</a>
 
-             <li class = "nav-item">
-              <%= link_to "Profile[#{current_user.username.upcase}]",user_path(current_user),class:"btn" %>
-            </li>
-             <% if @redirect_to_page_id %>
-                <div class = "action-dropdown">
+    <button class="navbar-toggler" 
+            type="button"
+            data-bs-toggle="collapse"
+            data-bs-target="#navbarSupportedContent"
+            aria-controls="navbarSupportedContent"
+            aria-expanded="false"
+            aria-label="Toggle navigation">
 
-                    <li><a class = "nav-item" href="<%= new_bucket_path %>">Create Bucket</a></li>
+      <span class="navbar-toggler-icon"></span>
+    </button>
 
-                    <li><hr class =  "dropdown-divider"></li>
```

Comment:
try to put each html tag on it's own line
insted of `<li><hr class = "dropdown-divider"></li>`
`<li>
  <hr class = "dropdown-divider">
</li>`
`hr` tag is nested inside li `tag` with 2 spaces or 1 tab

also very very important
![Screenshot from 2024-02-29 23-00-11](https://github.com/IssahBen/cinema/assets/62436291/f63818af-be66-45cf-8013-e5f4b2b746f1)
you should always close a tag on the same distance where it was opened
ESPECIALLY when you use if else end statements, otherwise it's almost impossible to read and if you come back to it a couple month later you would not understand where is the `if` statement ends, whats inside, is this `else` belong to this `if` or that and so on